.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 python:
   - "2.7"
+  - "3.6"
```

Comment:
Maybe we can remove 2.7. Will reduce a bit of CI busywork if we just force everyone to use a particular version of Python. :D

Reply:
Removed 2.7 as well :)